sphinx/source/developers/cryptography.rst

```diff
@@ -47,6 +47,5 @@ Authenticated encryption in CCF relies on AES256-GCM. Ledger authentication reli
 Public-key certificates, signatures, and ephemeral Diffie-Hellman key exchanges all rely on
 elliptic curves. They can be configured to use one of the following implementations:
 
- * ed25519 from `project Everest <https://project-everest.github.io/>`_.
```

Comment:
`ed25519` is not supported with the current version of mbedtls that we've got. I think it's better to just remove it from our docs altogether.